admin-dev/themes/new-theme/js/pages/order/create/create-order-page.js

```diff
@@ -488,4 +562,32 @@ export default class CreateOrderPage {
 
     this.cartEditor.changeCartAddresses(this.cartId, addresses);
   }
+
+  /**
+   * Refresh addresses list
+   *
+   * @param {boolean} refreshCartAddresses optional
+   *
+   * @private
+   */
+  refreshAddressesList(refreshCartAddresses) {
+    const cartId = $(createOrderMap.cartBlock).data('cartId');
+    $.get(this.router.generate('admin_carts_info', {cartId})).then((cartInfo) => {
+      this.addressesRenderer.render(cartInfo.addresses);
+
+      if (refreshCartAddresses) {
+        this.changeCartAddresses();
+      }
+    }).catch((e) => {
+      window.showErrorMessage(e.responseJSON.message);
```

Comment:
@NeOMakinG `showErrorMessage()` => `window. showErrorMessage()` is that right ?

Reply:
What do you mean ? Didn't understand this one